lib/galaxy/web/framework/base.py

```diff
@@ -539,9 +539,9 @@ def send_file(start_response, trans, body):
         start = None
         end = None
         if trans.request.range:
-            start = trans.request.range.start
-            end = trans.request.range.end
-            file_size = trans.response.headers["content-length"]
+            start = int(trans.request.range.start)
+            file_size = int(trans.response.headers["content-length"])
+            end = int(file_size if end is None else trans.request.range.end)
```

Comment:
I'm not sure this is quite correct, if `trans.request.range.end` is not sent or not parsed correctly we should probably respond with a 416. I'll take a look at what the media player is sending.

Reply:
When I originally encountered the error, request.range.end was None.  I think it's sending "0-" as the range string.

Reply:
I did a quick check on what's valid for the Range header and having an open-ended range is a legitimate request 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range